src/cms/page-home-new.md

```diff
@@ -12,6 +12,16 @@ You can maintain a selection of different home pages, and activate the page that
 
 1. Expand ![Expansion selector]({% link images/images/btn-expand.png %}) the **Default Pages** section and set **CMS Home Page** to the new page.
 
+   Other created pages can be used as the home page, such as:
```

Comment:
I don't think we need this addition. It can be removed from the PR.

Reply:
@dobooth 
Thanks, done ✅